Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -910,13 +912,17 @@ def audit_revision_and_version_scheme
           newest_committed_version ||= previous_version
           newest_committed_checksum ||= previous_checksum
           newest_committed_revision ||= previous_revision
+          newest_committed_url ||= stable.url
         end
 
         break if previous_version && current_version != previous_version
       end
 
+      pypi_prefix = "https://files.pythonhosted.org/packages/"
```

Comment:
```suggestion
      pypi_prefix = PyPI:: PYTHONHOSTED_URL_PREFIX
```

Reply:
I had thought about unprivating and using that constant from `utils/pypi`, but I didn't want to include the entire file just for the constant.

Reply:
Is this a significant performance drop? I don't think the PyPI module is very large. In my opinion, it's better to have consistency and not repeat this hardcoded value anywhere.

Reply:
You're right that it's not that big of a module, I'll go ahead and include and use it.